lib/rubocop/config.rb

```diff
@@ -172,8 +173,16 @@ def base_dir_for_path_parameters
 
     private
 
+    def validate_section_presence(name)
+      return unless @hash.key?(name) && @hash[name].nil?
+      fail ValidationError,
+           "empty section #{name} found " \
```

Comment:
I wonder if we can make this message a bit more informative (more text, maybe include line number).